opentracing-web-servlet-filter/src/main/java/io/opentracing/contrib/web/servlet/filter/TracingFilter.java

```diff
@@ -90,18 +95,20 @@ public TracingFilter() {
      * @param tracer
      */
     public TracingFilter(Tracer tracer) {
-        this(tracer, Collections.singletonList(ServletFilterSpanDecorator.STANDARD_TAGS));
+        this(tracer, Collections.singletonList(ServletFilterSpanDecorator.STANDARD_TAGS), null);
     }
 
     /**
      *
      * @param tracer tracer
      * @param spanDecorators decorators
+     * @param skipPattern null or pattern to exclude certain paths from tracing e.g. "/health"
      */
-    public TracingFilter(Tracer tracer, List<ServletFilterSpanDecorator> spanDecorators) {
+    public TracingFilter(Tracer tracer, List<ServletFilterSpanDecorator> spanDecorators, Pattern skipPattern) {
```

Comment:
It could be handled by that way, but makes it more ackward. However in this case the skip pattern is less likely to be used, so having it optional may be better?

Yes the rules could set via system property/env var.